src/tests/mocks/account_mocks.cairo

```diff
@@ -106,14 +112,28 @@ mod CamelAccountMock {
     #[event]
     #[derive(Drop, starknet::Event)]
     enum Event {
+        #[flat]
```

Comment:
is this change being properly tested?

Reply:
The `utils:pop_log` method will panic with `'Event has extra keys'` if events are not flattened.